data_collection/gazette/spiders/mg_belohorizonte.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, datetime, timedelta
+
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+    name = "mg_belo_horizonte"
+    allowed_domains = ["portal6.pbh.gov.br"]
+    documents_url = (
+        "http://portal6.pbh.gov.br/dom/iniciaEdicao.do"
+        "?method=DomDia&dia={}&comboAno={}"
+    )
+    start_date = date(2002, 3, 21)
+
+    def start_requests(self):
+        current_date = date.today()
+        while current_date >= self.start_date:
+            formatted_date = current_date.strftime("%d/%m/%Y")
+            yield Request(
+                self.documents_url.format(formatted_date, current_date.year),
+                callback=self.parse_page_link,
+                meta={"current_date": current_date},
+            )
+            current_date -= timedelta(days=1)
+
+    def parse_page_link(self, response):
+        xpath = '//p[contains(@class,"dom-chamadas")]/a/@href'
+        url_path = response.xpath(xpath).extract_first()
+        if url_path:
+            yield Request(
+                response.urljoin(url_path),
+                callback=self.parse_details,
+                meta={"current_date": response.meta["current_date"]},
+            )
+
+    def parse_details(self, response):
+        xpath = '//a[contains(@alt, "link_anexo")]/@href'
+        file_url = response.xpath(xpath).extract_first()
+        if file_url:
+            yield Gazette(
+                date=response.meta["current_date"],
+                file_urls=[response.urljoin(file_url)],
+                is_extra_edition=False,
+                territory_id=self.TERRITORY_ID,
+                power="executive_legislature",
```

Comment:
```suggestion
                power="executive_legislature",
```

Esse valor causa um error de validação: `{'power': ['Value not in choices']}`